src/Screens/CreateCommon/CreateStep4.js

```diff
@@ -94,6 +100,125 @@ const CreateStep4 = props => {
     });
   };
 
+  const ipfsUpload = async formData => {
+    return await IpfsClient.addAndPinString(
+      JSON.stringify({
+        name: formData.name,
+        byline: formData.byline,
+        description: formData.description,
+        courseOfAction: formData.action,
+        mainValue1: formData.funding,
+        mainValue2: formData.minimum,
+        mainValue3: 'empty value',
+      }),
+    );
+  };
+
+  const forgeCommon = async _ipfsHash => {
```

Comment:
Let's update this code following the model here: https://github.com/daostack/common-scripts/blob/master/createCommon.js